features/feature_case/math/math-bfloat16.cu

```diff
@@ -0,0 +1,37 @@
+// ====------ math-bfloat16.cu---------- *- CUDA -* ----===////
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+// ===----------------------------------------------------------------------===//
+
+#include <cuda.h>
+#include <cuda_bf16.h>
+#include <iostream>
+
+__device__ bool valuesAreClose(float a, float b, float epsilon = 0.05f) {
+  return (fabs(a - b) < epsilon);
+}
+
+__global__ void testMathFunctions(char *const TestResults) {
+  const auto bf16 = __float2bfloat16(3.14f);
+  const float f32 = __bfloat162float(bf16);
```

Comment:
Please add comment to tell the reason on:  why here not check exactly same data, but check the data is very close.